src/backend/distributed/shared_library_init.c

```diff
@@ -1462,6 +1463,17 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL | GUC_NOT_IN_SAMPLE,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_single_shard_table_multi_node_placement",
+		gettext_noop("Enables placement of single shard distributed tables in"
+					 " all nodes of the cluster"),
+		NULL,
+		&EnableSingleShardTableMultiNodePlacement,
+		false,
```

Comment:
Since we want to backport this, I think it indeed makes sense for this to be false by default. But lets create a PR right after merging this to change the default to true for future releases.